src/frontend/api/axios.ts

```diff
@@ -54,7 +54,7 @@ export const loggFeil = (
     innloggetSaksbehandler?: ISaksbehandler,
     feilmelding?: string
 ) => {
-    if (process.env.NODE_ENV === 'development') {
+    if (process.env.NODE_ENV !== 'development') {
```

Comment:
Var dette en bug?

Reply:
Nei, var for å ikke skape for mye støy under utvikling. Systemet har jo egentlig ikke blitt brukt noe særlig :)